daq/host.py

```diff
@@ -768,7 +770,7 @@ def _get_module_params(self):
             'local_ip': ext_loip,
             'target_ip': self.target_ip,
             'target_mac': self.target_mac,
-            'target_port': str(self.target_port) if self.target_port else None,
+            'target_port': str(self.target_port) if self.target_port else self.nonof_dev_port,
```

Comment:
another code smell here -- you're duplicating the logic.  If anywhere, this should go in the constructor, and keep these occurrences as self.target_port and replace OTHER uses of self.target_port with a more appropriate value that induces the relevant behavior.